cursorless-talon/src/targets/target_types.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from typing import Literal, Optional, Union
+
+
+@dataclass
+class PrimitiveTarget:
+    type = "primitive"
+    mark: Optional[dict]
+    modifiers: Optional[list[dict]]
```

Comment:
Looks like you stopped drilling down the dataclasses here?  Seems reasonable

Reply:
Year I think so. As we talked about yesterday these are never actually never read Talon side.